clouds/snowflake/modules/sql/processing/_DELAUNAYHELPER.sql

```diff
@@ -6,36 +6,51 @@ CREATE OR REPLACE FUNCTION @@SF_SCHEMA@@._DELAUNAYHELPER
 (inputPoints ARRAY)
 RETURNS ARRAY
 AS $$ (
-    WITH points AS (
-        SELECT ARRAY_AGG(unnested.VALUE) AS arrayPoints,
-            MIN(ST_X(TO_GEOGRAPHY(unnested.VALUE))) AS xMin,
-            MIN(ST_Y(TO_GEOGRAPHY(unnested.VALUE))) AS yMin,
-            MAX(ST_X(TO_GEOGRAPHY(unnested.VALUE))) AS xMax,
-            MAX(ST_Y(TO_GEOGRAPHY(unnested.VALUE))) AS yMax,
-            ABS(MAX(ST_X(TO_GEOGRAPHY(unnested.VALUE))) - MIN(ST_X(TO_GEOGRAPHY(unnested.VALUE)))) AS xExtent,
-            ABS(MAX(ST_Y(TO_GEOGRAPHY(unnested.VALUE))) - MIN(ST_Y(TO_GEOGRAPHY(unnested.VALUE)))) AS yExtent
-        FROM LATERAL FLATTEN(input => inputPoints) AS unnested
-      ), voronoi AS (
-      SELECT @@SF_SCHEMA@@._VORONOIHELPER(
-            arrayPoints, 
-            ARRAY_CONSTRUCT(xMin - xExtent * 0.5, 
-                            yMin - yExtent * 0.5, 
-                            xMax + xExtent * 0.5, 
+    WITH distinct_rounded_points AS (
+        SELECT ST_POINT(x, y) AS point FROM (
+          SELECT DISTINCT ROUND(ST_X(point), 5) AS x, ROUND(ST_Y(point), 5) AS y
+          FROM (
+            SELECT TO_GEOGRAPHY(pointjson.VALUE) AS point
+            FROM LATERAL FLATTEN(input => inputPoints) AS pointjson
```

Comment:
I would try to keep it snake case as in the rest of the function.
`inputPoints` -> `input_points`
`pointjson`-> `point_json`

Reply:
Also `arrayPoints`. Actually I see that we have kind of a mix in the variable names in this function. I see xMin, xMax... as well.